### ISSUE TITLE
Minnowboard Turbot sdcard booting

### ISSUE THREAD

**Issue by daniellejeong** [MinnowBoard-org/website#168]
Hi,
I've encountered some problems when trying to boot the minnowboard,
so I am looking for ways to fix it.

When I power the board, it displays the message 
'?bytes of system memory bytes checked' 
and shows the message "boot failed" twice.
Then it enters a UEFI interactive shell where it says,

UEFI Interactive Shell v2.1
EDK II
UEFI v2.40 (EDK II, 0x00010000)
map: No mapping found.
Press ESC in 1 seconds to skip startup.nsh or any other key to continue.
Shell>

Since the booting order is ubuntu->Fedora->EFI Internal Shell,
I'm assuming that Ubuntu and Fedora booting are failing.

I tried to locate the boot stub as one expert recommended, 
(at FS0:\EFI\BOOT\bootx64.efi or FS1:\)
but any command related to memory/directory gives me error.
Does this mean that the SATA hard drive is broken?

After trying for a while, I'm now attempting to put the minnowboard turbot image 
on the sdcard to boot it with Ubuntu 14.04 operating system.
I found a page that explains the process of loading system image to sdcard for minnowboard Max
(https://software.intel.com/en-us/node/564892).
My second question is, is Minnowboard Max image compatible with minnowboard Turbot?

I don't have much experience working with boards,
so please do understand if I may be asking a silly question.
Any help or suggestions would be really great!

Thanks.

**Comment by zwei4** [comment]
UEFI Interactive Shell v2.1
EDK II
UEFI v2.40 (EDK II, 0x00010000)
map: No mapping found.
Press ESC in 1 seconds to skip startup.nsh or any other key to continue.
Shell>

Above message shows that there is no SD card detected on your board.
Please make sure your SD card functions well.

**Comment by zwei4** [comment]
Sorry, i was meant to say your SATA hard disk is not detected.

**Comment by daniellejeong** [comment]
I was working with another Minnowboard Turbot that was working properly,
and then I encountered the same problem again!!!

I was having trouble with the network configuration,
so what I did was,
I shut down the computer, switched the network card to another one,
turned on the computer, and then found that the problem wasn't solved,
so I put back the original network card again.

When I did that and reboot the system,
it again started to show me the message "boot failed" and then
entered the UEFI shell!

Did I do something wrong?

**Comment by ebrent8** [comment]
This issue was posted in the website bugs.  **I think you need to post this in the project Issues section which is monitored for this kind of support.**

https://github.com/MinnowBoard-org/bugs-and-help/issues

**Comment by ebrent8** [comment]
@daniellejeong  Were you able to get an answer for this question?  If so, let's close the issue.  If not, please post it under the **bugs and help** link above where you can get assistance.  Thx.

**Comment by zwei4** [comment]
Is there any Blk and FS showed by UEFI Shell this time?

**Comment by daniellejeong** [comment]
ebrent8 i will close the issue because I kind of solved it :)
and hi zwei4! 
I concluded that my SATA drive is broken because it couldn't locate any directories in the UEFI shell.
So I put the image file in the SD card and was able to boot it!

Another new Minnowboard Turbot that was showing the same problem was just my silly mistake.
I thought I was switching the network chip, but what I actually did was, I switched the SATA drive because they looked similar. So when I switched back the original one, there was some loose contact that the board couldn't recognize the SATA drive. But then after a while, it started to work again!

But thank you for your advises and help!!
